README.md

```diff
@@ -233,6 +233,29 @@ fastify.listen({ port: 3000 }, err => {
 ```
 
 Note: Fastify's `onError` and error handlers registered by `setErrorHandler` will still be called for errors encountered *before* the websocket connection is established. This means errors thrown by `onRequest` hooks, `preValidation` handlers, and hooks registered by plugins will use the normal error handling mechanisms in Fastify. Once the websocket is established and your websocket route handler is called, `fastify-websocket`'s `errorHandler` takes over.
+
+### Custom preClose hook:
+
+By default, all ws connections are closed when the server closes. If you wish to modify this behaviour, you can pass your own `preClose` function.
+
+Note that `preClose` is responsible for closing all connections and closing the websocket server.
+
+```js
+const fastify = require('fastify')()
+
+fastify.register(require('@fastify/websocket'), {
+  preClose: (done) => {
```

Comment:
This should also support async functions (without the callback).

Reply:
Added a note to clarify that it indeed supports them (it supports whatever Fastify's preClose hook supports)